docs/en/about/news.md

```diff
@@ -2,217 +2,216 @@
 sidebar: auto
 ---
 
-# 大史记
+# The Great Chronicles
 
 ## 2024.9.12
 
-大史记重出江湖？！
+The Great Chronicles Return to the Scene?!
 
 ## 2024.9.7 <Badge>[v24.9.7](https://github.com/XTLS/Xray-core/releases/tag/v24.9.7)</Badge>
 
-更改版本号之后的首次发版。
+First release after changing the version number.
 
-- 这次移除了 QUIC 以及 DomainSocket 传输，移除了两处远古配置遗留代码。
-  - 二进制大小比 v1.8.24 减小了 1MB。
-- 依然有每次必备的 bug 修复。
+- This time, QUIC and DomainSocket transports were removed, along with two pieces of legacy code.
+  - The binary size is 1MB smaller than v1.8.24.
+- As always, there are essential bug fixes.
 
 ## 2024.8.30 <Badge>[v1.8.24](https://github.com/XTLS/Xray-core/releases/tag/v1.8.24)</Badge>
 
-在等待 SplitHTTP multiplex controller 期间，main 分支已经积累了大量重要更新，所以我们决定先发一个版本。
+While waiting for the SplitHTTP multiplex controller, the main branch had accumulated many important updates, so we decided to release a version first.
 
-- Socks 入站现在默认兼容 HTTP 代理请求。
+- The Socks inbound now supports HTTP proxy requests by default.
 - UDP noise (preview)
-- 还有一些改进。
+- And some other improvements.
 
-由于传统版本号的存在，为每个版本规划功能、进行排期已经严重阻碍了新功能的开发、合并、发布。所以我们决定从下个版本开始弃用传统的版本号，改用发版日期作为版本号，如 v24.8.30，并取消版本规划，全面采用流式更新，写好的功能直接合并，不再等待，预计每月月底发一个版本。
-毕竟对于反审查软件来说，相较于传统的版本号，新功能的及时性、每月更新更为重要，而不是发一个功能确定的版本并长期维护。
-下个版本会移除一些历史久远的代码，以后日常积累新代码、提醒迁移，跨年新版删代码、breaking。
-我们相信有了各位的捐款以及对发版形式的革新，Xray-core 这个项目会发展得更好。
+Due to the existence of traditional version numbers, planning features and scheduling for each release have severely hindered the development, merging, and release of new features. Therefore, we decided to abandon traditional version numbers starting with the next release and use the release date as the version number, such as v24.8.30, and cancel version planning, fully adopting continuous updates. Features will be merged and released as soon as they are ready, with a version released at the end of each month.
+
+After all, for anti-censorship software, the timeliness of new features and monthly updates are more important than a feature-defined version with long-term maintenance.
```

Comment:
After all, as a software aims to help people bypass censorship, instead of maintaining a long-term stable version, it's more important to adapt new features and keep updating monthly.